shub/config.py

```diff
@@ -81,11 +82,12 @@ def load(self, stream):
                 yaml_option_conf = yaml_cfg.get(option, {})
                 option_conf.update(yaml_option_conf)
                 if option == 'images' and yaml_option_conf:
-                    click.echo(
+                    warnings.warn(
```

Comment:
Another option is to monkeypatch https://docs.python.org/3/library/warnings.html#warnings.showwarning